website/content/v1.8/talos-guides/configuration/disk-management.md

```diff
@@ -0,0 +1,190 @@
+---
```

Comment:
also link this to release notes

Reply:
I will do this once I have something configurable (in the next PR before beta)